pkg/repo/index.go

```diff
@@ -135,8 +135,13 @@ func (i IndexFile) Add(md *chart.Metadata, filename, baseURL, digest string) {
 
 // Has returns true if the index has an entry for a chart with the given name and exact version.
 func (i IndexFile) Has(name, version string) bool {
-	_, err := i.Get(name, version)
-	return err == nil
+	vs := i.Entries[name]
```

Comment:
Should there be a check here to ensure `i.Entries` has key `name`?

Reply:
If it's not present, the iteration should iterate zero times and return false which is the behaviour we want:

https://play.golang.org/p/UrJS0jt108P